config/stage.json

```diff
@@ -102,7 +102,7 @@
 		"onboarding/import-redirect-to-themes": true,
 		"p2/p2-plus": true,
 		"page/export": true,
-		"pattern-assembler/add-pages": false,
```

Comment:
@Automattic/lego  Just an FYI for future CfTs and internal testing, we shouldn't put test features on staging because all of a8c uses staging.  I explained a bit more here: p1699483518899059/1699330091.581499-slack-C048CUFRGFQ

😄